### PR TITLE
HW2: Add project link to smh2278.md, rebase with main

### PR DESCRIPTION
**Pull request by sebastianhereu**
Update smh2278.md

added md files

Update smh2278.md

Update fundy.md

Update fundy.md

Update scrapy.md

Update scrapy.md

Update smh2278.md

### CODE REVIEW

File: students/smh2278.md [hunk 1]
```diff
@@ -4,18 +4,16 @@ Name: Sebastian Hereu
...
 UNI: smh2278
 
-Github: [link](https://github.com/sebastianhereu)
+|  |  | 
+|:--|:--|
+|Name|Sebastian Hereu|
+|UNI| smh2278|
+| GitHub Profile | [link](https://github.com/sebastianhereu) |
+| GitHub README | [link](https://github.com/sebastianhereu/sebastianhereu/blob/main/README.md) |
+| project link (fundy) | [link](https://github.com/sebastianhereu/fundy) |
```
Comment (timkpaine) [inline students/smh2278.md:13]: move into project markdown file

File: students/smh2278.md [hunk 2]
```diff
@@ -4,18 +4,16 @@ Name: Sebastian Hereu
...
 
 ## Project Proposal
-
-Idea:
-
-1) Fundamental Analysis library - takes time series data and provides indicators like RSI and Kaufman Efficiency Ratio. Can also trigger trades to a
-specified API with certain events i.e. RSI crosses some value
+- [fundy](./projects/python/fundy.md)
```
Comment (timkpaine) [inline students/smh2278.md:16]: update link path and re-squash 😈

File: students/smh2278.md [hunk 3]
```diff
@@ -4,18 +4,16 @@ Name: Sebastian Hereu
...
 
 ## Project Contributions
-
-1) Arrow library - It is a friendly library that basically works with dates and times
-2) Numpy
-3) Delorean - simplifying for DateTime
-4) Requests
+- [scrapy](./projects/python/scrapy.md)
```
Comment (timkpaine) [inline students/smh2278.md:19]: update link path and re-squash 😈

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: students/smh2278.md (hunk 2 above)
Comment (sebastianhereu) [reply students/smh2278.md]: Is my solution fine?

I made my fixes, committed them, and then ran 

git reset --soft "HEAD^"
git commit --amend

to change history and ammend with the original commit from rebasing.

File: students/smh2278.md (hunk 1 above)
Comment (sebastianhereu) [reply students/smh2278.md]: Done.

**Pull request merged**